sdks/go/examples/cookbook/combine/combine.go

```diff
@@ -63,11 +63,16 @@ type extractFn struct {
 	MinLength int `json:"min_length"`
 }
 
+// A global context for simplicity.
+var ctx = context.Background()
+
 func (f *extractFn) ProcessElement(row WordRow, emit func(string, string)) {
+	small_words := beam.NewCounter("example.namespace", "small_words")
```

Comment:
For perf reasons it's better to declare the counter in the package scope, but either works correctly.

Better to show the recommended path as part of real examples or recipes I think.

Reply:
Sounds good. @herohde pointed out the WC example uses counters too so I'm going to go take a look at that as well.